app/controllers/users_controller.rb

```diff
@@ -0,0 +1,33 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def create
+    # @user = User.new(user_name: params[:user_name], email: params[:email], password: params[:password])
```

Comment:
### No Commented Code
you can have commented brief explanation but no commented code, if you want to check out our previews code use git commits history